commands/inventory/list_agents.go

```diff
@@ -60,15 +73,13 @@ func (res *listAgentsResult) String() string {
 type listAgentsCommand struct {
 }
 
-func getAgentStatus(s *string, disabled bool) string {
-	res := strings.ToLower(pointer.GetString(s))
+// This is used in the json output. By convention, statuses must be in uppercase
+func getAgentStatus(status *string) string {
+	res := pointer.GetString(status)
 	if res == "" {
 		res = "unknown"
 	}
-	if disabled {
-		res += " (disabled)"
-	}
-	return res
+	return strings.ToUpper(res)
```

Comment:
We don't need to make uppercase data from server because they are already in uppercase.